ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -526,6 +526,17 @@
 {%         if router_bgp.address_family_evpn.next_hop_unchanged is arista.avd.defined(true) %}
 - Next-hop-unchanged is explicitly configured (default behaviour)
 {%         endif %}
+{%         if router_bgp.address_family_evpn.next_hop_mpls_resolution_ribs is arista.avd.defined() %}
+{%             set rib_position = ["Primary", "Secondary", "Tertiary" ] %}
+{%             for rib in router_bgp.address_family_evpn.next_hop_mpls_resolution_ribs %}
+{%                 if rib.rib_type == "tunnel-rib-colored" %}
+{%                     set rib_type = "tunnel-rib colored system-colored-tunnel-rib" %}
+{%                 else %}
+{%                     set rib_type = rib.rib_type %}
+{%                 endif %}
+- Next-hop MPLS resolution {{ rib_position[loop.index - 1] }}-RIB : {{ rib_type }} {{ rib.rib_name | arista.avd.default('') }}
```

Comment:
I think you have to append the rib_name in a separate `elif` above. Otherwise you will add an extra space at the end of the generated CLI/doc.

Reply:
```suggestion
- Next-hop MPLS resolution {{ rib_position[loop.index0] }}-RIB : {{ rib_type }} {{ rib.rib_name | arista.avd.default('') }}
```